js/hw4.js

```diff
@@ -0,0 +1,226 @@
+// task1
+let Car;
+
+function makeCar(producer, model, production_year, avg_speed) {
+    let car = {
+        producer: producer,
+        model: model,
+        production_year: production_year,
+        avg_speed: avg_speed,
+
+        printCar() {
```

Comment:
It is ok to do it that way, but it's more like `class-oriented` way of creating methods
Since it's an object - it can be like that:

`printCar: () => 'Авто має модель ${this.model}, було виготовлено ${this.producer} у ${this.production_year} та має середню швидкість ${this.avg_speed} км/год.'` (or function declaration)

Reply:
fixed